chain/Cargo.toml

```diff
@@ -43,7 +43,7 @@ thiserror = "1.0.37"
 
 # wasm
 #[target.'cfg(all(target_arch = "wasm32", not(target_os = "emscripten")))'.dependencies]
-wasm-bindgen = { version = "=0.2.82", features = ["serde-serialize"] }
+wasm-bindgen = { version = "=0.2.83", features = ["serde-serialize"] }
```

Comment:
This version is required for using serde wasm bindgen dependency which we added a bit ago to avoid potential cyclical dependency issues.